### DISCUSSION TITLE
[UI] The Top Menu style and behavior feedback

### DISCUSSION THREAD

**Discussion by MerryPanda** [phcode-dev/phoenix#1616] [General]
The Top Menu (File, Edit, Find, View...) font size is changed with the theme font size change, which is strange and unexpeced.

The Top Menu should be better separated from the text area, imho. Now they blend together. In Brackets it was not alligned to text area and had dark color, which looked and felt much better.

The Top Bar (Windows App top bar) is not colored.

**Comment by abose** [comment]
Ah yes, the notch situation in Windows in light theme mode. The current theme is optimized more for dark theme in windows. We have a major redesign of the light theme scheduled for july, so that should address this.
